OTKit/otkit-colors/token.yml

```diff
@@ -2,29 +2,29 @@ props:
   # Primary Action
   # This color is only used on objects, backgrounds with large text (24px or larger, bold), and will be used as a color to guide users towards actionable content.
   # =============================================
-  color-primary:
+  color-brand:
     value: "#da3743"
   color-primary-active:
```

Comment:
then this must also be changed; based on the changes you've made, I would guess something like "color-brand-lighter", or "color-brand-2", or something.

Reply:
if the color brand is only one, why not having just color-brand?

Reply:
I'm not talking about `color-brand`, I'm talking about `color-primary-active`

Reply:
oh I see, sorry, I actually agree with you @lastquestion